### PR TITLE
Stats page to new format

### PR DESCRIPTION
**Pull request by makinde**
Along the way I realized that I rewrote a bunch of stuff, but might have lost sight about if things are better. I let you guys weigh in on if it's worth it. If not, we can roll this back and the diff before it. I /think/ it'll make it easier to build a player page and do more with the data going forward.

- Move stats.js to calculate stuff in getStaticProps. This essentially moves the logic of generateData.js into getStaticProps and various util methods.
- Make player info into an object so we can store other info in there (notably, [fb]id's). This will allow us to grab profile pics, and are better url safe handles for player profiles.
- "Fix" (I think) the asc/desc logic for sorting stats. The old arrows didn't match my understanding of asc/desc, but I could be wrong.
- Lots of lodash usage to avoid rewriting those functions. Lmk if this is bad. I think it makes the code a bit easier to read.

### CODE REVIEW

File: constants/index.js [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
 export const PAR = 27;
+
+export const PLAYERS = [
+  {
+    id: 'eric',
+    spreadsheetKey: 'eric g',
+    name: 'Eric Giovanola',
+  },
+  {
+    id: 'zamore',
```
Comment (ericgio) [inline constants/index.js:11]: Whoa, "zamore", "fzamore", and "ericzamore" all redirect to the profile page! I didn't realize you could do that...

File: data/utils.js [hunk 2]
```diff
@@ -1,65 +1,138 @@
-function normalizeName(name) {
-  name = name.toLowerCase();
+import {
+  round, groupBy, sumBy, sortBy, orderBy, filter, first, last, isEmpty,
+  zipObject,
+} from 'lodash';
```
Comment (ericgio) [inline data/utils.js:4]: If this doesn't fit on one line, each item should be on a separate line.

File: pages/stats.js [hunk 3]
```diff
@@ -1,61 +1,37 @@
 import cx from 'classnames';
 import moment from 'moment-timezone';
 import React, { useCallback, useState } from 'react';
+import {
+  orderBy, filter, find, mean, values, sortBy, min, map, round as roundTo,
+} from 'lodash';
```
Comment (ericgio) [inline pages/stats.js:6]: Same here. One line or separate lines like the import from `data/utils` below.

**Review by ericgio — APPROVED** [review]

File: data/utils.js (hunk 2 above)
Comment (makinde) [reply data/utils.js:4]: Gotcha. Will change. Was surprised that the linter didn't force it/correct it, so assumed this was your pref.

you may want to configure this setting in the project's/your eslint config: https://eslint.org/docs/rules/object-curly-newline

Comment (ericgio) [reply data/utils.js]: That rule actually doesn't work as desired here: https://stackoverflow.com/questions/58656094/eslint-rule-to-put-a-new-line-inside-import

There was a proposal to address that, but it was never merged: https://github.com/eslint/eslint/issues/9259#issuecomment-525289402

**Pull request merged**